src/client/utils/storage/ServerCookieStorage.ts

```diff
@@ -11,9 +11,8 @@ export class ServerCookieStorage implements MinimalStorage {
       return this.changedItems[item]
     }
 
-    return decodeURIComponent(
-      this.requestCtx.cookies.get(item, { signed: false })!,
```

Comment:
Interesting to note is the TS would've complained if we hadn't told it not to with the `!`